topic-operator/src/main/java/io/strimzi/operator/topic/BatchingTopicController.java

```diff
@@ -292,10 +292,15 @@ private static Either<TopicOperatorException, Boolean> validateUnchangedTopicNam
     }
 
     private PartitionedByError<ReconcilableTopic, Void> createTopics(List<ReconcilableTopic> kts) {
+        Map<ReconcilableTopic, TopicOperatorException> newTopicsErrors = new HashMap<>();
         var newTopics = kts.stream().map(reconcilableTopic -> {
-            // Admin create
```

Comment:
Was this comment relevant?

Reply:
I think it is a leftover, as the admin call is few lines below with a clear log line.